doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
+## Methods:
+- `StartBroadcasting`
+- `StopBroadcasting`
+- `Connect`
+- `Disconnect`
+- `KillConnection`
+
+Events:
+- `peerAvailabilityChanged`
+- `networkChanged`
+
+***
+
+METHODS:
+
+***
+
+### `StartBroadcasting(deviceName, portNumber, callback)`
+
+This method starts advertising or broadcasting its availability at the following port that PouchDB/Express-PouchDB is listening.  
```

Comment:
Is it advertising or broadcasting? We should pick one. I would suggest a more action oriented description. Something like "This method instructs the native layer to advertise the availability of the device under the specified deviceName and to direct any incoming connections to the specified portNumber available on localhost over TCP/IP."

I don't specifically mention PouchDB/Express-PouchDB because it doesn't really matter what is listening. Just so long as it speaks TCP/IP.